docs/source/RunningTestsRemotely.rst

```diff
@@ -174,6 +174,22 @@ image (``ldoktor/fedora-avocado``) in the default image repository
     ...
     Status: Downloaded newer image for docker.io/ldoktor/fedora-avocado:latest
 
+Use custom docker images
+------------------------
+
+One of possible ways to develop avocado is to create docker image with your devel tree.
+This is good way to test your devel branch w/o breaking system instalation ::
+
+    Fetch git source as usual
```

Comment:
Please either comment this line, or preferrably, make this the introduction to the following commands.  What I suggest is something like:

"This is a good way to test your devel branch without breaking your system instalation.  To do so, start by fetching the source from Git as usual::

```
   git clone github.com/avocado-framework/avocado.git avocado.git
   cd avocado.git
```

Then, optionally, make changes to your tree::

```
  patch -p1 < MY_PATCH
```

And so on...